src/data/init/init-common-anims.ts

```diff
@@ -2,15 +2,14 @@ import { LegacyAnimConfig } from "#animations/anim-config";
 import { commonAnims } from "#animations/common-anims";
 import { globalScene } from "#app/global-scene";
 import { CommonAnim } from "#enums/common-anim";
-import { getTSEnumKeys, getTSEnumValues } from "#utils/common-utils";
 
 export function initCommonAnims(): Promise<void> {
   return new Promise((resolve) => {
-    const commonAnimNames = getTSEnumKeys(CommonAnim);
-    const commonAnimIds = getTSEnumValues(CommonAnim);
+    const commonAnimNames = Object.keys(CommonAnim);
+    const commonAnimIds = Object.values(CommonAnim);
```

Comment:
You can avoid the `const commonAnimId: CommonAnim = ...` type definition by providing a type argument here

```suggestion
    const commonAnimIds = Object.values<CommonAnim>(CommonAnim);
```

Reply:
Hm, I'm not sure why I put the `: CommonAnim` there in the first place, removing it doesn't seem to break anything.